data/achievements-view.ui

```diff
@@ -128,7 +128,7 @@
                   </object>
                   <packing>
                     <property name="name">GRID</property>
-                    <property name="title" translatable="yes">page0</property>
+                    <property name="title" translatable="no">page0</property>
```

Comment:
I know this text doen't appear in the UI, but I wonder if it's used by screen readers for accesibility. For now let's go with this solution, making it untranslatable.